requirements/testing.txt

```diff
@@ -4,13 +4,13 @@ django-dynamic-fixture==2.0.0
 
 # 3.6.1 and >3.2.5 is incompatible
 # with pytest-describe 0.11.0
-pytest==3.2.5
-pytest-django==3.1.2
-pytest-describe==0.11.0
-pytest-xdist==1.22.0
+pytest==3.6.2
+pytest-django==3.3.2
+pytest-describe==0.11.1
+pytest-xdist==1.22.2
 apipkg==1.4
 execnet==1.5.0
-Mercurial==4.4.2
+Mercurial==4.6.1
```

Comment:
This is terrible :/

> Mercurial 4.3 and newer require Python 2.7. Mercurial versions up to 4.2 support Python versions 2.6 to 2.7, provided they include the following standard library components: 

https://www.mercurial-scm.org/wiki/SupportedPythonVersions

Reply:
I'm confused how we're even installing `hg` in our test suite if 3.x isn't supported. It seems this shouldn't work?

Reply:
I think we weren't hitting any of the Py 3.x problems but it wasn't supported. Although, it was possible to be installed and used:

> Mercurial is actively being ported to Python 3. As of Mercurial 4.3, some commands work on Python 3. However, Python 3 is not yet a supported platform. 

(from the bottom of the page I linked)